### PR TITLE
Implement maximize on macOS (closes #89)

### LINKED ISSUE #89: Implement maximizing on Mac

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @AlexanderMoskovkin @georgiy-abbasov

### CODE REVIEW

File: src/natives/get-window-max-bounds/mac/get-window-max-bounds.m [hunk 1]
```diff
@@ -0,0 +1,111 @@
+//
+//  shot_window.m
+//  Screenshot
```
Comment (georgiy-abbasov) [inline src/natives/get-window-max-bounds/mac/get-window-max-bounds.m]: Change comments please

File: src/natives/get-window-max-bounds/mac/get-window-max-bounds.m [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    } 
+
+    return NULL;
+}
+
+struct WindowRect getScreenRect (NSScreen *screen, NSScreen* primaryScreen, int useVisibleFrame) {
+    NSRect screenRect        = useVisibleFrame ? [screen visibleFrame] : [screen frame];
+    NSRect primaryScreenRect = [primaryScreen frame];
+
+    double convertedY = primaryScreenRect.size.height - screenRect.origin.y - screenRect.size.height;
```
Comment (georgiy-abbasov) [inline src/natives/get-window-max-bounds/mac/get-window-max-bounds.m]: What's logic behind this ?

File: src/api/maximize.js [hunk 3]
```diff
@@ -1,9 +1,41 @@
...
+        array:  boundsArray,
+        left:   boundsArray[0],
+        top:    boundsArray[1],
+        right:  boundsArray[2],
+        bottom: boundsArray[3]
+    };
+}
+
+async function getWindowBounds (windowId, bundleId) {
+    return getBoundsFromString(await exec(`osascript -e 'tell app id "${bundleId}" to get bounds of window id ${windowId}'`));
```
Comment (AlexanderMoskovkin) [inline src/api/maximize.js]: It's hard to read when you use await function call as an argument for another function. Please do it separately

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "testcafe-browser-tools",
-  "version": "1.2.0",
+  "version": "1.3.0",
```
Comment (AlexanderMoskovkin) [inline package.json]: I suppose it can be 1.2.1

**Pull request merged**